azure-quantum/tests/unit/test_pasqal.py

```diff
@@ -17,47 +17,85 @@
 # Will update once paspal 
-#@pytest.mark.pasqal
-#@pytest.mark.live_test
-#class TestPasqalTarget(QuantumTestBase):
-#    """Tests the azure.quantum.target.Pasqal class."""
-#
-#    def _run_job(
-#        self,
-#        pulser: str,
-#        input_params: Union[InputParams, Dict[str, Any], None],
-#    ) -> Optional[Result]:
-#        workspace = self.create_workspace()
-#
-#        target = Pasqal(workspace=workspace, name=PasqalTarget.SIM_EMU_TN)
-#        job = target.submit(
-#            input_data=pulser,
-#            name="qdk-python-test",
-#            input_params=input_params,
-#        )
-#
-#        job.wait_until_completed(timeout_secs=DEFAULT_TIMEOUT_SECS)
-#        job.refresh()
-#
-#        job = workspace.get_job(job.id)
-#        self.assertTrue(job.has_completed())
-#
-#        return Result(job)
-#
-#    def test_job_submit_pasqal_typed_input_params(self) -> None:
-#        num_runs = 200
-#        result = self._run_job(TEST_PULSER, InputParams(runs=num_runs))
-#        self.assertIsNotNone(result)
-#        self.assertEqual(sum(v for v in result.data.values() if isinstance(v, int)), num_runs)
-#
-#    def test_job_submit_pasqal_dict_input_params(self) -> None:
-#        num_runs = 150
-#        result = self._run_job(TEST_PULSER, input_params={"runs": num_runs})
-#        self.assertIsNotNone(result)
-#        self.assertEqual(sum(v for v in result.data.values() if isinstance(v, int)), num_runs)
-#
-#    def test_job_submit_pasqal_default_input_params(self) -> None:
-#        default_num_runs = 100
-#        result = self._run_job(TEST_PULSER, None)
-#        self.assertIsNotNone(result)
-#        self.assertEqual(sum(v for v in result.data.values() if isinstance(v, int)), default_num_runs)
+@pytest.mark.pasqal
+@pytest.mark.live_test
+class TestPasqalTarget(QuantumTestBase):
+   """Tests the azure.quantum.target.Pasqal class."""
+
+   def _run_job(
+       self,
+       pulser: str,
+       input_params: Union[InputParams, Dict[str, Any], None],
+   ) -> Optional[Result]:
+       workspace = self.create_workspace()
+
+       target = Pasqal(workspace=workspace, name=PasqalTarget.SIM_EMU_TN)
+       job = target.submit(
+           input_data=pulser,
+           name="qdk-python-test",
+           input_params=input_params,
+       )
+
+       job.wait_until_completed(timeout_secs=DEFAULT_TIMEOUT_SECS)
+       job.refresh()
+
+       job = workspace.get_job(job.id)
+       self.assertTrue(job.has_completed())
+
+       return Result(job)
+
+   def test_job_submit_pasqal_typed_input_params(self) -> None:
+       num_runs = 200
+       result = self._run_job(TEST_PULSER, InputParams(runs=num_runs))
+       self.assertIsNotNone(result)
+       self.assertEqual(sum(v for v in result.data.values() if isinstance(v, int)), num_runs)
+
+   def test_job_submit_pasqal_dict_input_params(self) -> None:
+       num_runs = 150
+       result = self._run_job(TEST_PULSER, input_params={"count": num_runs})
```

Comment:
@kikomiss, t it seems that Pascal targets now expect `count` option instead of `runs` (At least on prod).